### PR TITLE
Feat: UX scaling updates

### PR DESCRIPTION
**Pull request by jimhunty**
Scaling updates specifically to Comments Button and Article Topics
https://nidigitalsolutions.jira.com/browse/REPLAT-3322

### CODE REVIEW

File: packages/article-topics/package.json [hunk 1]
```diff
@@ -62,6 +62,7 @@
     "webpack-cli": "2.1.4"
   },
   "dependencies": {
+    "@times-components/context": "0.4.6",
```
Comment (craigbilner) [inline packages/article-topics/package.json]: need to bump

File: packages/button/src/button.js [hunk 2]
```diff
@@ -5,9 +5,8 @@ import transformTitle from "./utils";
 import { propTypes, defaultProps } from "./button-prop-types";
 import styles from "./styles";
 
-const Button = ({ onPress, style, title }) => {
+const Button = ({ onPress, style, textStyle, title }) => {
```
Comment (craigbilner) [inline packages/button/src/button.js]: ideally we shouldn't allow arbitrary styling of components and have a concrete api

File: packages/article-topics/src/article-topics.js [hunk 3]
```diff
@@ -1,18 +1,44 @@
...
+import styleguide from "@times-components/styleguide";
 import ArticleTopic from "./article-topic";
 import styles from "./styles";
 import propTypes from "./article-topic-prop-types";
 
 const { style: ViewPropTypesStyle } = ViewPropTypes;
 
+const renderArticleTopics = (topics, onPress, fontStyle) =>
+  topics.map(({ name, slug }) => (
+    <ArticleTopic
```
Comment (craigbilner) [inline packages/article-topics/src/article-topics.js]: same re:style api on `Button`

Comment (jimhunty) [reply packages/article-topics/src/article-topics.js]: @craigbilner This is following the change in technical direction that was used in Key Facts. Do you have a suggestion? Im not a fan of the above approach.

Comment (craigbilner) [reply packages/article-topics/src/article-topics.js]: :speaking_head: @tuncaulubilge ideally a component shouldn't be arbitrarily styled and instead look something like this if possible:

```
<Component
  fontWeight={400}
/>

// component.js

styles=[styles.componentBaseStyles, {fontWeight}]
```

Comment (jimhunty) [reply packages/article-topics/src/article-topics.js]: So for the above example, we would have a separate prop for font size and line height?

**Pull request merged**